module-1/kubernetes/deployment.yaml

```diff
@@ -0,0 +1,19 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: test-app-deployment
+spec:
+  replicas: 3
+  selector:
+    matchLabels:
+      app: test-app
+  template:
+    metadata:
+      labels:
+        app: test-app
+    spec:
+      containers:
+      - name: test-app-container
+        image: ghcr.io/akashsara/test-app:latest 
```

Comment:
which platform are you using? 
![image](https://github.com/user-attachments/assets/a3359fac-4cb4-4600-adb1-1130c8fc2c88)

Reply:
Windows! Maybe I'll do something to make it OS-independent and set up stuff for Win/Mac/Linux separately.